velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -363,23 +357,23 @@ TEST_F(DateTimeFunctionsTest, fromUnixtimeWithTimeZone) {
         evaluate("from_unixtime(c0, '+01:00')", makeRowVector({unixtimes}));
 
     auto expected = makeTimestampWithTimeZoneVector(
-        makeFlatVector<int64_t>(
-            size, [&](auto row) { return unixtimeAt(row) * 1'000; }),
-        makeConstant((int16_t)900, size));
+        size,
+        [&](auto row) { return unixtimeAt(row) * 1'000; },
+        [](auto row) { return 900; });
```

Comment:
nit: `[](auto /*row*/) { return 900; }` to suppress linter complaint about unused variable.

Reply:
Fixed